src/IROperator.cpp

```diff
@@ -2654,4 +2654,18 @@ Expr gather(const std::vector<Expr> &args) {
     return make_scatter_gather(args);
 }
 
+Expr extract_bits(Type t, const Expr &e, const Expr &lsb) {
+    return Internal::Call::make(t, Internal::Call::extract_bits, {e, lsb}, Internal::Call::Intrinsic);
+}
+
+Expr concat_bits(const std::vector<Expr> &e) {
+    user_assert(!e.empty()) << "concat_bits requires at least one argument\n";
```

Comment:
Should this method require at least two arguments? Either that, or a call with one argument should just simplify to that argument (in Simplify_Call.cpp perhaps?)